.github/workflows/gradle.yml

```diff
@@ -0,0 +1,19 @@
+name: Java CI
+
+on: [push]
```

Comment:
This should include pull-request as well.

Reply:
Good idea, thanks